docs/overview/00_gen.md

```diff
@@ -0,0 +1,24 @@
+
+# Olympus was created to be a better medium of exchange
```

Comment:
I think this header should be shortened to fit on one line 
<img width="295" alt="image" src="https://user-images.githubusercontent.com/80423742/234569075-9bb4eaa6-352f-4960-b555-8147fa708e8f.png">

Reply:
alternatively you can add a label on line 1 of the file like

```
---
sidebar_label: Why Olympus Protocol
---
```